exercises/practice/forth/src/forth.chpl

```diff
@@ -0,0 +1,3 @@
+module {{ExerciseNamespace}} {
```

Comment:
@kytrinyx since we are at it, you may also want to double check this (same in `.meta/reference.chpl`)

Reply:
Oops. Yes :) I will check that too.